spotinst_sdk2/clients/ocean_cd/__init__.py

```diff
@@ -0,0 +1,950 @@
+    __base_oceancd_cluster_url = "https://api.spotinst.io/ocean/cd/cluster"
+    __base_oceancd_vp_url = "https://api.spotinst.io/ocean/cd/verificationProvider"
+    __base_oceancd_vt_url = "https://api.spotinst.io/ocean/cd/verificationTemplate"
+    __base_strategy_url = "https://api.spotinst.io/ocean/cd/strategy"
+    __base_rollout_spec_url = 'https://api.spotinst.io/ocean/cd/rolloutSpec'
+    __base_rollout_url = 'https://api.spotinst.io/ocean/cd/rollout'
+    __base_workload_url = 'https://api.spotinst.io/ocean/cd/workload'
+    ENTITY_NAME1 = 'oceancd_cluster'
+    ENTITY_NAME2 = 'oceancd_verification_provider'
+    ENTITY_NAME3 = 'oceancd_verification_template'
+    ENTITY_NAME4 = 'oceancd_strategy'
+    ENTITY_NAME5 = 'oceancd_rollout_spec'
+    ENTITY_NAME6 = 'oceancd_rollout'
+    ENTITY_NAME7 = 'oceancd_workloads'
+
+    def get_oceancd_cluster(self, cluster_id: str):
+        """
+        Get an existing Ocean CD cluster.
+
+        # Arguments
+        cluster_id (String): OceanCD Cluster ID
+
+        # Returns
+        (Object): OceanCD Cluster API response
+        """
+        response = self.send_get(
+            url=self.__base_oceancd_cluster_url + "/" + cluster_id,
+            entity_name=self.ENTITY_NAME1)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+
+    def get_all_oceancd_clusters(self):
+        """
+        List all Ocean CD clusters.
+
+        # Returns
+        (List): List of Ocean CD CLuster API response
+        """
+        response = self.send_get(
+            url=self.__base_oceancd_cluster_url,
+            entity_name=self.ENTITY_NAME1)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"]
+
+    def update_oceancd_cluster(self, cluster_notification: oceancd_cluster.ClusterNotification, cluster_id: str):
+        """
+        Update Ocean CD cluster notification settings.
+
+        # Arguments
+        cluster_id (String): OceanCD Cluster ID
+        node_update (StatefulNode): StatefulNode Object
+
+        # Returns
+        (Boolean): Response Status
+        """
+        request = oceancd_cluster.UpdateClusterNotification(cluster_notification)
+
+        excluded_node_update_dict = self.exclude_missing(
+            json.loads(request.toJSON()))
+
+        formatted_node_update_dict = self.convert_json(
+            excluded_node_update_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_node_update_dict)
+
+        response = self.send_put(
+            body=body_json,
+            url=self.__base_oceancd_cluster_url + "/" + cluster_id,
+            entity_name=self.ENTITY_NAME1)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["status"]
```

Comment:
Shouldn't return status only

Reply:
We are only getting status as response for this api